src/components/ChangeTag/ChangeTag.tsx

```diff
@@ -0,0 +1,33 @@
+import { ArrowDownIcon, ArrowUpIcon } from '@chakra-ui/icons'
+import { Skeleton } from '@chakra-ui/react'
+import type { TagProps } from '@chakra-ui/tag'
+import { Tag } from '@chakra-ui/tag'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import { Amount } from 'components/Amount/Amount'
+
+export type ChangeTagProps = {
+  value: string | number | undefined
```

Comment:
naming suggestion for sanity, so we ensure we consistently pass the correct unit for percentage (i.e. we could wrongly pass 50 instead of 0.5 to mean 50%):

```suggestion
  percentChangeDecimal: string | number | undefined
```

Reply:
you're right, fixed it in [84220ed](https://github.com/shapeshift/web/pull/7124/commits/84220ed98cefd7aff9faf6549a3e07a55df59dce)